unlock-app/src/__tests__/config.test.js

```diff
@@ -96,12 +96,7 @@ describe('config', () => {
       expect.assertions(2)
       expect(config.requiredNetwork).toEqual('Dev')
       expect(config.providers).toMatchObject({
-        HTTP: {
-          connected: false,
-          headers: undefined,
-          host: 'http://127.0.0.1:8545',
-          timeout: 0,
-        },
+        HTTP: 'http://127.0.0.1:8545',
```

Comment:
```suggestion
        HTTP: getWeb3Provider('http://127.0.0.1:8545'),
```

This way, if we do decide to actually return a provider from this function, the test has a fighting chance of passing (and also makes it clearer what we are doing here)